### PR TITLE
Allow defining FEEL Popup container

### PR DESCRIPTION
**Pull request by pinussilvestrus**
Depends on https://github.com/bpmn-io/properties-panel/pull/280
Related to https://github.com/camunda/team-hto/issues/344
Required by https://github.com/camunda/form-playground/pull/101

This allows to provide a feel popup container via config.

```js
new FormPlayground({
  data,
  schema,
  propertiesPanel: {
    feelPopupContainer: rootRef.current
  },
});
```

Demo: https://feel-popup-root--camunda-form-playground.netlify.app/

### CODE REVIEW

File: packages/form-js-editor/rollup.config.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ export default [
         targets: [
           { src: 'assets/form-js-editor-base.css', dest: 'dist/assets' },
           { src: '../../node_modules/@bpmn-io/draggle/dist/dragula.css', dest: 'dist/assets' },
-          { src: '../../node_modules/@bpmn-io/properties-panel/assets/properties-panel.css', dest: 'dist/assets' }
+          { src: '../../node_modules/@bpmn-io/properties-panel/dist/assets/properties-panel.css', dest: 'dist/assets' }
```
Comment (vsgoulart) [inline packages/form-js-editor/rollup.config.js:73]: Why is this necessary?

**Review by vsgoulart — APPROVED** [review]

File: packages/form-js-editor/rollup.config.js (hunk 1 above)
Comment (pinussilvestrus) [reply packages/form-js-editor/rollup.config.js:73]: There was a change in the core properties panel that now also stores the styles inside `dist` (for consistency reasons). It's not a breaking change as the old path is still there, but I found it better to follow the best practice sooner than later.

Comment (pinussilvestrus) [reply packages/form-js-editor/rollup.config.js:73]: https://github.com/bpmn-io/properties-panel/blob/main/CHANGELOG.md#340

**Review by Skaiir — APPROVED** [review]
Requires conflict resolutions due to pressing the button on groups I reckon 👀
